README.md

```diff
@@ -13,45 +13,85 @@ For Ubuntu users: [complete step-by-step development environment setup procedure
 ## Non-Node Dependencies
 
 * PostgreSQL 10+ with PostGIS
-* [OSRM](https://github.com/Project-OSRM/osrm-backend/)
-* [trRouting](https://github.com/chairemobilite/trRouting/)
+* [OSRM](https://github.com/Project-OSRM/osrm-backend/): It is the routing engine used by Transition, to calculate the routes for various modes: for example walking, cycling, driving, bus in urban setting, suburban bus, etc.
+* [trRouting](https://github.com/chairemobilite/trRouting/): An open source routing engine to calculate the route between an origin and a destination, or to calculate accessibility from/to a point, using public transit network. It is the main engine used for public transit simulations.
 * yarn: [debian/ubuntu](https://classic.yarnpkg.com/en/docs/install/#debian-stable) [macos](https://classic.yarnpkg.com/en/docs/install/#mac-stable)
-* Rust (it is used to run the json2capnp cache service which makes the application much faster if there's a lot of transit data)
+* Rust: It is used to run the json2capnp cache service which makes the application much faster if there's a lot of transit data.
 
 ## Installation
 
-* Install Node and Yarn
+* Install dependencies:
+```
+sudo apt-get install postgresql postgis lua5.3 liblua5.3-dev \
```

Comment:
This is ubuntu specific, it should be mentionned. (Something like an example for Ubuntu xx.xx)
This already does not work for Debian/bullseye, as lua5.4 is used.